### ISSUE TITLE
p5.Image mixes up image width and height attributes

### ISSUE THREAD

**Issue by SimonBuxx** [processing/p5.js#1666]
Today, I experimented with the p5.Image class. While debugging a piece of code that didn't work, I found out that the width and height attribute of my test image that was in portrait mode (464x568 Pixel) were switched (568x464 Pixel). It worked perfectly fine with a landscape image.
Here's my code:
```javascript
var img;

function preload() {
    img = loadImage("assets/image.jpg");
}

function setup() {
    createCanvas(800, 800);

    img.loadPixels();
    console.log("width: " + img.width + ", height: " + img.height);
}
```

The result is "width: 568, height: 464" altough it is a portrait picture.

**Comment by lmccart** [comment]
can you please post the image file you're using

**Comment by lmccart** [comment]
I'm unable to reproduce this, but please reopen if you can post an image file to test with.

**Comment by SimonBuxx** [comment]
I found out that it seems to work fine for every picture but the test picture (a pass photo of mine) that I used. The picture is 464 x 568px. If I open the picture in Paint and just save it, the size is recognized correctly. Very strange, but obviously not a common problem ;-) Maybe my image is just broken, altough Windows Explorer and Paint are showing the correct dimensions.

PS: Sorry that I didn't answer earlier, forgot about it.
